pkg/config/config.go

```diff
@@ -52,6 +52,18 @@ func NewConfig(path string) (*Config, error) {
 	if err := viper.BindEnv("httphandler.port"); err != nil {
 		return nil, err
 	}
+	if err := viper.BindEnv("clickup.CQYUVTWH2.webhooksecret"); err != nil {
+		return nil, err
+	}
+	if err := viper.BindEnv("clickup.CQYUVTWH2.token"); err != nil {
+		return nil, err
+	}
+	if err := viper.BindEnv("clickup.CQTG74FUH.webhooksecret"); err != nil {
+		return nil, err
+	}
+	if err := viper.BindEnv("clickup.CQTG74FUH.token"); err != nil {
+		return nil, err
+	}
```

Comment:
Please read from file. Do not hadcode `CQYUVTWH2` in go code

Reply:
done